src/Entity/Event.php

```diff
@@ -0,0 +1,348 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\EventRepository")
+ */
+class Event
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(type="string", length=1000)
+     */
+    private $shortDescription;
+
+    /**
+     * @ORM\Column(type="string",nullable=true)
+     */
+    private $longDescription;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $beginningDate;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $endDate;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $location;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="App\Entity\Country", inversedBy="events")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $country;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $priceMin;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $priceMax;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $priceCurrency;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $organizers;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $organizerContact;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $contactSite;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $contactFB;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $contactOther;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="EventType", inversedBy="events")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $type;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="EventCategory", inversedBy="events")
+     */
+    private $category;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $status;
+
+    /**
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private $publishDate;
```

Comment:
@daria-sieroshtan 
I think you want to use `Timestampable` behaviour here... don't you?

Reply:
Sure, but I'll think about implementation a little bit later, it is related to admin functionality.

Reply:
@daria-sieroshtan 
please add todo than (mb even remove property altogether for now)

Reply:
todo is added to general todo list.
No, this property is crucial even in current dummy state.

Reply:
@daria-sieroshtan 
It's a goood practise to leave TODOs in code, possibly with references to external tickets
But OK, leave it as is